### DISCUSSION TITLE
Doesn't work properly

### DISCUSSION THREAD

**Discussion by nitsc** [hacksider/Deep-Live-Cam#247] [Q&A] [answered]
Just like the picture shows, **Doesn't work properly**, how to solve it? Could you teach me, please?
![image](https://github.com/user-attachments/assets/a806e259-59b8-465d-af5a-413cb2994176)

**Comment by nitsc** [comment, marked as answer]
And it is too slowly!!! I think we can put up the speed.

**Comment by nitsc** [comment]
now, it change to such circumstances
![image](https://github.com/user-attachments/assets/84cf2995-1b35-4f91-b729-13fd23a4e12a)

**Reply by davidharket** [reply]
What did you do to fix the black box error?

**Reply by Ku-Tadao** [reply]
@davidharket https://github.com/hacksider/Deep-Live-Cam/discussions/242

**Reply by davidharket** [reply]
I believe this sadly is one of the 1%. Even though i am on python 3.10.00, i am still getting this error. I just tried creating another venv, but it still did not work. I have also tried installing and running without a venv. Still getting the black box error.

**Reply by nitsc** [reply]
> What did you do to fix the black box error?

It's just a performance problem with your computer, so it's recommended to change your computer or graphics card

**Reply by nitsc** [reply]
> I believe this sadly is one of the 1%. Even though i am on python 3.10.00, i am still getting this error. I just tried creating another venv, but it still did not work. I have also tried installing and running without a venv. Still getting the black box error.

I think there is something wrong with this project, there are errors in many of the usage commands. For example, after the command to use DirectX graphics card acceleration, the official is -directml, which should actually be -dml

**Reply by Ku-Tadao** [reply]
> I believe this sadly is one of the 1%. Even though i am on python 3.10.00, i am still getting this error. I just tried creating another venv, but it still did not work. I have also tried installing and running without a venv. Still getting the black box error.

what bootup args did you use, maybe i can do a check, as i only know about Cuda as of now.
